packages/sourcecred/src/core/ledger/ledger.js

```diff
@@ -518,7 +531,13 @@ export class Ledger {
     if (!this._accounts.has(id)) {
       throw new Error(`identity ${id} not found`);
     }
-    const {active} = this.account(id);
+    const {active, payoutAddresses} = this.account(id);
+    if (this._externalCurrency) {
```

Comment:
It seems like there is a possible path where you can disableAccounting while still having an externalCurrency defined. This seems undesireable for this particular line of code. It seems to me like you want tight coupling between disabling accounting and having an external currency. If this is true, I'm wondering if you should maybe combined the actions together, specifically merge disable/enableAccounting into set/removeExternalCurrency. If there is a reason to have them separate, please elaborate.

Reply:
@blueridger I originally had a tight coupling, but realized, that an external currency might be desirable for reporting or other reasons. The external currency is needed to find the correct payout address for any distribution. The upshot of this is that even something as simple as the CSV integration depends upon the external currency being set.

I also intentionally am allowing for our extant use case (the one we've been running up until this point in  theSourceCred community) where the ledger serves as an off-chain tracker for IOUs (accounting enabled) while allowing for payouts onchain that can then be processed in the ledger automatically. This basically means that Seth's/the multisig's current workflow would be simplified because the ledger updates would be automated if we decided to conitnue to payout in USDC. This story basically is ideal for smaller communities that want to pay folks in crypto but don't yet want to handle the overhead of deploying a smart contract and/or setting up their own token.